README.md

```diff
@@ -0,0 +1,26 @@
+# Project Theia
+
+## What is it?
```

Comment:
Would it be more formal to say "What is Project Theia?"

Reply:
I don't think so. At least not when it's under the H1 of "Project Theia". Maybe if it part of an FAQ on a wiki who's H1 was just "Frequently Asked Questions".

I'm open to be convinced tho, if you want.